pipelinewise/cli/schemas/tap.json

```diff
@@ -232,6 +232,46 @@
         }
       },
       "required": ["type"]
+    },
+    "tap_github": {
+      "properties": {
+        "type": {
+          "enum": ["tap_github"]
+        },
+        "db_conn": {
```

Comment:
Can you add conditions so that either `repository` or `organization` is present?